app/views/api/backend_usages/index.html.slim

```diff
@@ -1,6 +1,8 @@
 h1 Backends
 p.intro For a Product to work, it needs to have at least one Backend with a Private Base URL (your API). If multiple Backends are added to a Product, each Backend should have a unique Public Path.
 
+= pf_button_in_title 'Add Backend', new_admin_service_backend_usage_path(@service)
```

Comment:
I haven't seen this method before, but maybe you can use [this one](https://github.com/3scale/porta/blob/master/app/helpers/buttons_helper.rb#L150), we are already using it in several places

Reply:
I created this method a while ago to encapsulate Patternfly's CTA. I think we should not mix "3scale style" with "PF style". From behind, it uses the method you're referring.

https://github.com/3scale/porta/search?q=pf_button_in_title